app/src/main/java/org/oppia/app/testing/HtmlParserTestActivity.kt

```diff
@@ -18,14 +18,25 @@ class HtmlParserTestActivity : InjectableAppCompatActivity() {
     setContentView(R.layout.test_html_parser_activity)
 
     val testHtmlContentTextView: TextView = findViewById(R.id.test_html_content_text_view)
-    val rawDummyString =
+    val rawDummyString1 =
       "\u003cp\u003e\"Let's try one last question,\" said Mr. Baker. \"Here's a pineapple cake cut into pieces.\"\u003c/p\u003e\u003coppia-noninteractive-image alt-with-value=\"\u0026amp;quot;Pineapple cake with 7/9 having cherries.\u0026amp;quot;\" caption-with-value=\"\u0026amp;quot;\u0026amp;quot;\" filepath-with-value=\"\u0026amp;quot;pineapple_cake_height_479_width_480.png\u0026amp;quot;\"\u003e\u003c/oppia-noninteractive-image\u003e\u003cp\u003e\u00a0\u003c/p\u003e\u003cp\u003e\u003cstrong\u003eQuestion 6\u003c/strong\u003e: What fraction of the cake has big red cherries in the pineapple slices?\u003c/p\u003e"
-    val htmlResult: Spannable =
+    val htmlResult1: Spannable =
       htmlParserFactory.create( /* entityType= */ "exploration", /* entityId= */ "oppia", /* imageCenterAlign= */ false)
         .parseOppiaHtml(
-          rawDummyString,
+          rawDummyString1,
           testHtmlContentTextView
         )
-    testHtmlContentTextView.text = htmlResult
+    testHtmlContentTextView.text = htmlResult1
+
+
+    val testHtmlContentOrderedListTextView: TextView = findViewById(R.id.test_html_content_with_ordered_list_text_view)
+    val rawDummyString2 = getString(R.string.faq_answer_1)
```

Comment:
Could we maybe instead add a test string that's similar to the FAQ string? Prefer not depending on production strings unless you're testing that specific string (otherwise if we change the string, we may break this completely unrelated test that's trying to verify something specific about html parsing).

Reply:
Added dummy string for testing.